.eslintrc.json

```
@@ -5,7 +5,7 @@
         "node": true
     },
     "parserOptions": {
-        "ecmaVersion": 2018
+        "ecmaVersion": 13
```

Why `13`? That numbering system was dropped already at ES6 that became ES2015. It’s possible to google for “ES2018” to learn about what was new in that version, but if you see `13` you have to calculate the 20XX year and then look it up.

---

I confess I picked the first thing google said would work. 

I needed it because the only way to get both flow and prettier to accept my class definitions is public class fields (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields).

```suggestion
        "ecmaVersion": 2022
```